{
  "contest": 372,
  "handle": "Mrxstrange",
  "risk": "2",
  "title": "The function will return 'returndata' indicating success even in cases of failure",
  "issueId": 324,
  "issueUrl": "https://github.com/code-423n4/2024-04-renzo-validation/issues/324"
}